CHANGE_LOG

```diff
@@ -1,3 +1,19 @@
+v0.23.2
+-------
+
+This is a bug fix release to assist in addressing a critical bug in Numba:
```

Comment:
"This is a bug fix release to assist in addressing a critical Numba issue that can affect users who download llvmlite packages from sources other than PyPI (pip), Anaconda, or Intel Python:"

Just to make it clear that this doesn't affect a large number of people.

Reply:
good idea, thanks, done.